### DISCUSSION TITLE
forward commits to one remore repo to another remote repo

### DISCUSSION THREAD

**Discussion by tomyang03** [community/community#43616] [Other Feature Feedback, Questions, & Ideas]
Hello, I'm looking for advice on how to accomplish this task.

I have 2 repositories named frontend and backend. 

I have another repository for docker that houses the code for the frontend and backend. 

I would like to set up something such that when there is a commit to either frontend or backend remote repositories, the other remote repository for docker will detect the changes and get the latest code into the repository for docker.  I have seen an approach where a remote URL for docker is added to the local frontend and backend repo, such that the local docker repo will get the latest code.  But what  I want is for whenever there is a code change to the remote frontend and backend, the remote docker repository will get the latest code.

**Comment by PedroDRodrigues** [comment]
You could set up a remote named "upstream" in the docker repository that points to the frontend and backend repositories. Then you can use the Git command "git pull upstream" to pull the latest code from the frontend and backend repositories into the docker repository. You could set up a script to run this command periodically (e.g. using cron) or trigger it manually whenever you want to update the docker repository with the latest code.

**Comment by tomyang03** [comment]
is there any way I could forward those commits to the remote docker repo without having to pull them to a local repo?

**Comment by PedroDRodrigues** [comment]
I dont think so! The only way I know to do it is doing this commits.
